software/base/src/test/java/org/apache/brooklyn/entity/software/base/test/mysql/DynamicToyMySqlEntityBuilder.java

```diff
@@ -88,18 +87,15 @@ public static final String dir(Entity e) {
     public static String getOsTag(Entity e) {
         // e.g. "osx10.6-x86_64"; see http://www.mysql.com/downloads/mysql/#downloads
         OsDetails os = ((SshMachineLocation)Iterables.getOnlyElement(e.getLocations())).getOsDetails();
-        if (os == null) return "linux-glibc2.5-x86_64";
+        if (os == null) return "linux2.6-x86_64";
         if (os.isMac()) {
-            String osp1 = os.getVersion()==null ? "osx10.8" //lowest common denominator
-                : new ComparableVersion(os.getVersion()).isGreaterThanOrEqualTo(OsVersions.MAC_10_9) ? "osx10.9"
-                : "osx10.8";  //lowest common denominator
             if (!os.is64bit()) {
                 throw new IllegalStateException("Only 64 bit MySQL build is available for OS X");
             }
-            return osp1+"-x86_64";
+            return "osx10.6-x86_64";
```

Comment:
Because the version 5.5.37 (or at least the package name) is only available for version 10.6 onwards. They probably removed the OSX version distinction.